geniza/footnotes/migrations/0016_rename_typed_texts.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 3.2.8 on 2022-02-10 18:12
+
+from django.conf import settings
+from django.db import migrations
+
+
+def rename_typed_texts(apps, schema_editor):
+    Source = apps.get_model("footnotes", "Source")
+    ContentType = apps.get_model("contenttypes", "ContentType")
+    LogEntry = apps.get_model("admin", "LogEntry")
+    source_contenttype = ContentType.objects.get(app_label="footnotes", model="source")
+    User = apps.get_model("auth", "User")
+    (user, _) = User.objects.get_or_create(
+        username=settings.SCRIPT_USERNAME, is_staff=False, is_active=False
+    )  # generic user for script execution (should already be created)
+
+    # get all sources called "typed texts"
+    typed_texts = Source.objects.filter(title="typed texts")
```

Comment:
any reason not to do a bulk / queryset update? is it because of the log action?

Reply:
That was the idea—maybe it makes more sense to do that and have a separate loop for the log entries since all of them will be identical except for the PK?

Reply:
yeah, I think I've done something like that before... 

this is the example I found, not exactly analogous https://github.com/Princeton-CDH/mep-django/blob/main/mep/footnotes/migrations/0007_rename_pulrbsc.py

Reply:
Thanks, that was extremely helpful! It's nearly identical. Done in 56396f0 👍